considerations.md

```diff
@@ -1,7 +1,8 @@
 # Extensibility
 
-Implementations that are reading/processing [manifests](manifest.md) or [image indexes](image-index.md) MUST NOT generate an error if they encounter an unknown property.
-Instead they MUST ignore unknown properties.
+Implementations storing or copying content MUST NOT generate error if they encounter an unknown manifest media type (e.g.: [image manifest](manifest.md)) or an unknown property in a manifest.
```

Comment:
"MUST NOT generate error if they encounter an unknown manifest" implies the `MANIFEST_UNKNOWN` error code in distribution spec is no longer valid. I don't think we can do that since it's not how any registry I'm aware of works today, everyone validates the manifest media type.

I would like to say "Implementations storing or copying content MUST NOT modify or alter the content in a way that would change the digest of the content."

Reply:
🙏 thank you for the feedback, @sudo-bmitch. Will #1043 merge today or early next week so I can rebase and incorporate these changes?

Reply:
That's a question for the other maintainers. We can merge once we get two approvals.

Reply:
I've pulled this out into a separate PR, replacing the line with yours